docs/zh_cn/04-supported-codebases/mmdet.md

```diff
@@ -210,7 +210,8 @@ cv2.imwrite('output_detection.png', img)
 |       [RepPoints](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/reppoints)       |    ObjectDetection    |      N      |    Y     |  N   |   ?   |    Y     |
 |            [DETR](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/detr)            |    ObjectDetection    |      Y      |    Y     |  N   |   ?   |    Y     |
 |       [CenterNet](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/centernet)       |   Object Detection    |      Y      |    Y     |  N   |   ?   |    ?     |
-| [Cascade Mask R-CNN](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/cascade_rcnn) | InstanceSegmentation  |      Y      |    N     |  N   |   N   |    Y     |
+|       [RTMDet](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/rtmdet)             |   Object Detection    |      Y      |    Y     |  N   |   ?   |    ?     |
```

Comment:
Could you double the supported models with model configs in mmdet.yml, and update mmdet.yml as well.
https://github.com/open-mmlab/mmdeploy/blob/6c26e887d4300c3afb57a0900c6ad53798d9dfda/tests/regression/mmdet.yml#L168-L353

Reply:
Of course, I'll update as soon as possible